src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockCropDead.java

```diff
@@ -140,6 +143,12 @@ public int quantityDropped(IBlockState state, int fortune, Random random)
         return count;
     }
 
+    @Override
+    public boolean canBlockStay(World world, BlockPos pos, IBlockState state) {
```

Comment:
```suggestion
    public boolean canBlockStay(World world, BlockPos pos, IBlockState state)
    {
```
Code style people... code style